Emoji messages are no longer being posted to #emoji_meta

## Description
We're getting a error when trying to schedule a message to be posted later in #emoji_meta

```
2021-06-24T17:54:51.359050+00:00 app[worker.1]: [2021-06-24 17:54:51,356] {app.py:67} ERROR - Failed to schedule message to channel emoji_meta: :strawberry_turtle_2:  (strawberry_turtle_2) has been added
2021-06-24T17:54:51.359084+00:00 app[worker.1]: Traceback (most recent call last):
2021-06-24T17:54:51.359085+00:00 app[worker.1]:   File "app.py", line 63, in send_emoji_message
2021-06-24T17:54:51.359085+00:00 app[worker.1]:     web_client.chat_scheduleMessage(**message)
2021-06-24T17:54:51.359086+00:00 app[worker.1]:   File "/app/.heroku/python/lib/python3.6/site-packages/slack/web/client.py", line 395, in chat_scheduleMessage
2021-06-24T17:54:51.359087+00:00 app[worker.1]:     return self.api_call("chat.scheduleMessage", json=kwargs)
2021-06-24T17:54:51.359088+00:00 app[worker.1]:   File "/app/.heroku/python/lib/python3.6/site-packages/slack/web/base_client.py", line 167, in api_call
2021-06-24T17:54:51.359088+00:00 app[worker.1]:     return self._event_loop.run_until_complete(future)
2021-06-24T17:54:51.359088+00:00 app[worker.1]:   File "/app/.heroku/python/lib/python3.6/asyncio/base_events.py", line 484, in run_until_complete
2021-06-24T17:54:51.359089+00:00 app[worker.1]:     return future.result()
2021-06-24T17:54:51.359089+00:00 app[worker.1]:   File "/app/.heroku/python/lib/python3.6/site-packages/slack/web/base_client.py", line 236, in _send
2021-06-24T17:54:51.359090+00:00 app[worker.1]:     return SlackResponse(**{**data, **res}).validate()
2021-06-24T17:54:51.359090+00:00 app[worker.1]:   File "/app/.heroku/python/lib/python3.6/site-packages/slack/web/slack_response.py", line 174, in validate
2021-06-24T17:54:51.359091+00:00 app[worker.1]:     raise e.SlackApiError(message=msg, response=self.data)
2021-06-24T17:54:51.359091+00:00 app[worker.1]: slack.errors.SlackApiError: The request to the Slack API failed.
2021-06-24T17:54:51.359096+00:00 app[worker.1]: The server responded with: {'ok': False, 'error': 'invalid_arguments', 'response_metadata': {'messages': ['[ERROR] must provide a number [json-pointer:/post_at]']}}
```

## Cause
The slack api has been updated and post_at should be a number.

https://api.slack.com/methods/chat.scheduleMessage#arg_post_at

--- a/emoji_message.py
+++ b/emoji_message.py
@@ -50,11 +50,11 @@ class EmojiMessage:
         return f':{self.icon_emoji}:  ({self.icon_emoji}) has been {self._get_past_tense_event()}'
 
     @staticmethod
-    def next_release_date(release_hour: int = 3) -> str:
+    def next_release_date(release_hour: int = 3) -> int:
         """Get release time as Unix EPOCH timestamp"""
         utc_time = datetime.datetime.utcnow()
         if utc_time.hour >= release_hour:
             utc_time += datetime.timedelta(days=1)
         send_timestamp = utc_time.replace(hour=release_hour, minute=0, second=0, microsecond=0,
                                           tzinfo=datetime.timezone.utc).timestamp()
-        return str(send_timestamp)
+        return int(send_timestamp)

